packages/react-components/src/components/AttachmentCard.tsx

```diff
@@ -77,68 +119,27 @@ export const _AttachmentCard = (props: _AttachmentCardProps): JSX.Element => {
     }
   }, [progress, showProgressIndicator, attachmentName, uploadStartedString, uploadCompletedString]);
 
-  const progressBarThicknessPx = 4;
-
   const containerClassName = mergeStyles({
     width: '12rem',
-    background: theme.palette.neutralLighter,
-    borderRadius: theme.effects.roundedCorner4,
-    border: `${_pxToRem(1)} solid ${theme.palette.neutralQuaternary}`,
-    cursor: 'pointer'
+    minWidth: '75%'
   });
 
-  const attachmentInfoWrapperClassName = mergeStyles({
-    padding: _pxToRem(12),
-    // To make space for the progress indicator.
-    paddingBottom: showProgressIndicator ? _pxToRem(12 - progressBarThicknessPx * 2) : _pxToRem(12)
-  });
-
-  const attachmentNameContainerClassName = mergeStyles({
-    paddingLeft: _pxToRem(4),
-    minWidth: '75%',
-    maxWidth: '75%'
-  });
-
-  const attachmentNameTextClassName = mergeStyles({
+  const fileNameContainerClassName = mergeStyles({
+    marginTop: _pxToRem(5),
+    width: '5.75rem',
```

Comment:
REM values will vary based on the default font size.
Should this be specified this way?

Reply:
Any other way of suggesting to achieve this? I tried eg. 50% width of parent and it will push the button all the way to the right
<img width="263" alt="image" src="https://github.com/Azure/communication-ui-library/assets/77021369/8d96d692-92b1-4094-a218-fc9e9c49af56">